resources/views/components/progress-bar/index.blade.php

```diff
@@ -1,23 +1,24 @@
 @props(['checkout' => true])
-<div v-cloak class="flex items-center space-x-[18px] mb-[5px] sm:mb-0 text-sm">
-    @if($checkout)
-        <span class="antialiased font-medium whitespace-nowrap text-inactive">Stap @{{ checkout.step }} van de 4</span>
-    @else
-        <span class="antialiased font-medium whitespace-nowrap text-inactive">Stap 4 van de 4</span>
-    @endif
-    @foreach(['Cart', 'Credentials', 'Payment', 'Confirmation'] as $stepTitle)
-        @if($checkout)
-            <div
-                v-on:click="goToStep({{ $stepTitle == 'Credentials' ? 2 : $loop->index }})"
-                class="hover:cursor-pointer"
-                :class="{'w-[12px] h-[12px] bg-accent': {{ $loop->index + 1 }} <= checkout.step, 'w-[12px] h-[12px] bg-border': {{ $loop->index + 1 }} > checkout.step, 'w-[20px] h-[20px] border-[4px] border-blue-110': {{ $loop->index + 1 }} === checkout.step}">
-            </div>
-        @else
+<div
+    class="flex items-center space-x-[18px] text-xs"
+    v-cloak
+>
+    <span class="whitespace-nowrap font-medium text-ct-inactive">
+        @lang('Step :step out of :total', [
+            'step' => '@{{ checkout.step }}',
+            'total' => count(config('rapidez.checkout_steps.' . config('rapidez.store_code')) ?? config('rapidez.checkout_steps.default')) - 1,
+        ])
+    </span>
+    @foreach (array_slice(config('rapidez.checkout_steps.' . config('rapidez.store_code')) ?? config('rapidez.checkout_steps.default'), 0, -1) as $stepTitle)
         <div
+            class="aspect-square w-3 cursor-pointer rounded"
             v-on:click="goToStep({{ $stepTitle == 'Credentials' ? 2 : $loop->index }})"
-            class="w-[12px] h-[12px] bg-blue w-[20px] h-[20px] border-[4px] border-blue-110 hover:cursor-pointer"
-            >
-            </div>
-        @endif
+            :class="{
+                'bg-ct-accent': {{ $loop->index + 1 }} <= checkout.step,
+                'bg-ct-border': {{ $loop->index + 1 }} > checkout.step,
+                'outline-4 outline outline-ct-accent-100': {{ $loop->index + 1 }} === checkout.step
```

Comment:
is this the only place where we use accent-100? if that's the case, i would like to remove the outline